integration-tests/README.md

```
@@ -0,0 +1,81 @@
+
+These variables let you point the test suite at non-default build or install locations.
+
+## Linting
+
+Before submitting a pull request, ensure you’ve run the linting commands below and have fixed all
+violations and suppressed all warnings.
+
+The following commands should be run from the integration-tests directory.
+
+To run ruff format checks:
+```shell
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)